src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -38,64 +38,67 @@ Above setting increases the TTL to 60 seconds.
 
-## Additional proxies
+## Additional forwards (formerly known as proxy) {#additional-forwards}
 
-The default proxy entry we set in CoreDNS is
+In CoreDNS version `1.4.0` the proxy plugin has been deprecated. The same behaviour can be achieved now with forward although syntax can be a bit different. The forward plugin indeed has better performance because it reuses opened upstream connections.
+
+The default forward entry we set in CoreDNS is
 
 ```yaml
-proxy . /etc/resolv.conf
+forward . /etc/resolv.conf
 ```
 
-You can add additional proxy entries by adding a each as a line to the proxy field of the user ConfigMap.
+You can add additional forward entries by adding a each as a line to the foward field of the user ConfigMap. They will be select by sequential order.
 
-For a single entry you can use the same line.
+You can use a simple line or multiple lines to define the upstreams of the default server block.
 
 ```yaml
 data:
-  proxy: foo.com 1.1.1.1
-
+  foward: 1.1.1.1
 ```
 
-For multplie entries you add a string with a proxy entry per line.
-
 ```yaml
 data:
-  proxy: |
-    foo.com 1.1.1.1
-    bar.com 8.8.8.8
+  foward: |
+    1.1.1.1
+    8.8.8.8
 ```
 
-Above example would result in following additional proxy entries in the CoreDNS configuration:
+__Warning:__ The number of forward upstreams is limited to 15.
+
+Above example would result in following additional forward entries in the CoreDNS configuration:
 
 ```yaml
-proxy foo.com 1.1.1.1
-proxy bar.com 8.8.8.8
+forward . 1.1.1.1 /etc/resolv.conf
 ```
 
-This setting would proxy all requests within foo.com to 1.1.1.1 which is Cloudflare's DNS and all requests within bar.com to 8.8.8.8 which is Google Public DNS. All other requests will be resolved by the default DNS provider set for your cluster.
+```yaml
+forward . 1.1.1.1 8.8.8.8 /etc/resolv.conf
+```
+
+This setting would forward all requests to 1.1.1.1 which is Cloudflare's DNS. If the first upstream fails the second IP (8.8.8.8) will be used as resolver. In case it fails too, all requests will be resolved by the default DNS provider set for your cluster.
 
-The proxy plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/proxy/).
+The forward plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/forward/).
 
 ## Advanced configuration
 
-In case you need to use an additional plugin or an existing plugin but with a special configuration, you can use the `custom` block in the configmap. It will be parsed directly into the Corefile.
+In case you need to have a finer granularity you can define custom server blocks with all configuration desired. They will be parsed after the catch-all block in the Corefile. As an example, let's define a block for a `example.com` with some custom configuration:
```

Comment:
```suggestion
In case you need to have a finer granularity you can define custom server blocks with all desired configuration. They will be parsed after the catch-all block in the Corefile. As an example, let's define a block for a `example.com` with some custom configuration:
```